sysl2/sysl/sysl.go

```diff
@@ -13,12 +13,17 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+// Version contains the sysl binary version
+// nolint:gochecknoglobals
```

Comment:
No whitespace before `nolint`. This is explicitly recommend by golangci-lint. https://github.com/golangci/golangci-lint#nolint
```go
//nolint:gochecknoglobals
```

Reply:
IDE formats automatically by adding a space. 😀
Should turn that off